test/react_test_utils_test.dart

```diff
@@ -91,58 +93,88 @@ testUtils({isComponent2: false, dynamic eventComponent, dynamic sampleComponent,
         event(domNode, eventData);
         expect(domNode.text, equals('$eventName $fakeTimeStamp'));
       });
+
+      if (expectEventType != null) {
+        test('with correct type', () {
+          SyntheticEvent capturedEvent;
+          final component = renderIntoDocument(div({
+            eventHandlerName: (e) => capturedEvent = e,
+          }));
+          event(react_dom.findDOMNode(component), eventData);
+          expectEventType(capturedEvent);
+        });
+      }
     }
 
     group('event', () {
-      group('animationEnd', () => testEvent(Simulate.animationEnd, 'animationEnd'));
-      group('animationIteration', () => testEvent(Simulate.animationIteration, 'animationIteration'));
-      group('animationStart', () => testEvent(Simulate.animationStart, 'animationStart'));
-      group('blur', () => testEvent(Simulate.blur, 'blur'));
+      group('animationEnd',
+          () => testEvent(Simulate.animationEnd, 'animationEnd', (e) => expect(e.isAnimationEvent, isTrue)));
```

Comment:
I just realized that these all test the `true`-cases for events, but not the `false` cases.

Could we update these tests to do something similar to event_helpers_test's `commonFalseTests`?

Reply:
Oh good catch! I'll update those!